src/Facebook/FacebookRequest.php

```diff
@@ -98,7 +98,7 @@ public function __construct(FacebookApp $app = null, $accessToken = null, $metho
     {
         $this->setApp($app);
         $this->setAccessToken($accessToken);
-        $this->setMethod($method);
+        $this->setMethod($method ?? '');
```

Comment:
Tohoto se trošku bojím, protože jsem to takto zkoušel při opravě testů a pak padalo na další chybě, že nedokáže parsovat nějakou url 🤔

Reply:
Btw tak to by už bylo lepší dát přímo do konstruktoru typehint na `string` a defaultní hodnotu na prázdný `string`, ne?

Reply:
Tam `null` bohuzel musi byt rozbije to testy. 😢 Oni tim nejak nastavuji defaultni request stav.

Reply:
Trosku jsem to upravil. driv to fungovalo takto 
<img width="288" alt="image" src="https://github.com/munipolis/php-graph-sdk/assets/39690283/351f1e90-79c8-41d5-a335-8c3652c3c822">

Reply:
vs ted 
<img width="1101" alt="image" src="https://github.com/munipolis/php-graph-sdk/assets/39690283/9891c313-43d2-4f4f-a65a-a7c301e93aa8">

Reply:
Takze funkcionalitu by to nemelo zmenit jen se nesmi nsateovat primo ten prazdny string pak pada test `testAMissingMethodWillThrow()`